app/assets/stylesheets/revised.scss

```diff
@@ -3,6 +3,7 @@
  * things from rails-assets
  *= require pikaday
  *= require selectize/selectize.bootstrap3.scss
+ *= require dataTables/bootstrap/3/jquery.dataTables.bootstrap
```

Comment:
Because we're using a .scss file instead of a .css file, the fancy import sprockets stuff is at the top, and then we import stuff using `@import` - this is because `@import` allows you to define variables/mixins in one file and use them in another, so we use it wherever we can.